Dockerfile.mysql

```diff
@@ -0,0 +1,13 @@
+FROM mysql:latest
```

Comment:
Using `mysql:latest` is not recommended for production environments as it may lead to compatibility issues when the image is updated. Consider specifying a specific version tag.